apis/metrics/metrics.go

```diff
@@ -38,7 +38,7 @@ var (
 	ImageActionsTimer = metrics.NewLabelTimer(subsystemPouch, "image_actions", "The number of seconds it takes to process each image action", "action")
 
 	// EngineVersion records the version and commit information of the engine process.
-	EngineVersion = metrics.NewLabelGauge(subsystemPouch, "engine", "The version and commit information of the engine process", "commit")
+	EngineVersion = metrics.NewLabelGauge(subsystemPouch, "engine", "The version and commit information of the engine process", "gitCommit", "pouchVersion", "kernelVersion")
```

Comment:
The name of labels is used by monitoring system, so it would be better to stay the same

Reply:
keep "commit", add "version" and "kernel"